backend/src/packages/projects/project.model.ts

```diff
@@ -1,16 +1,40 @@
+import { type RelationMappings, Model } from 'objection';
+
 import {
   AbstractModel,
   DatabaseTableName,
+  getJoinRelationPath,
 } from '~/libs/packages/database/database.js';
+import { FileModel } from '~/libs/packages/file/file.js';
 
 class ProjectModel extends AbstractModel {
-  public 'userId': number;
+  public 'userId': number | null;
+
+  public 'name': string | null;
+
+  public 'imageId': number | null;
 
-  public 'name': string;
+  public 'image': FileModel | null;
```

Comment:
should be `avatar` 
take the same approach as on user (for all, model, entity, type, etc)